airbyte-webapp/src/views/Connection/CatalogDiffModal/components/StreamRow.module.scss

```diff
@@ -39,14 +43,16 @@
   border-radius: variables.$border-radius-sm;
   padding: variables.$spacing-sm variables.$spacing-md;
   width: 226px;
-  opacity: 50%;
+  opacity: 0.5;
   background: colors.$red-100;
 }
 
-.nameCell {
+.nameCell,
+%nameCell {
   width: 140px;
   text-align: left;
-  & .row {
+
+  &.row {
```

Comment:
Was this an auto fix? because this is now actually a new selector? Was `.nameCell .row` beforehand and is now `.nameCell.row`.

Reply:
I manually changed this as I thought it was a typo. I'll revert.